src/Features/RedirectRedirectPages.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace P4\MasterTheme\Features;
+
+use P4\MasterTheme\Feature;
+
+/**
+ * @see description().
+ */
+class RedirectRedirectPages extends Feature {
+
+	/**
+	 * @inheritDoc
+	 */
+	public static function id(): string {
+		return 'redirect_redirect_pages';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	protected static function name(): string {
+		return __( 'Redirect redirect pages', 'planet4-master-theme-backend' );
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	protected static function description(): string {
+		return __(
+			'Not a typo: if turned on this will do a HTTP redirect from a tag redirect page URL to the tag URL, to avoid duplicate content.',
```

Comment:
A suggestion to make this sorter: _"Redirect from the Tags redirect pages to the tag URL, to avoid duplicate content."_

Reply:
I preserved "HTTP" as it makes it more obvious there's 2 different meanings.